source/MRViewer/MRObjectTransformWidget.cpp

```diff
@@ -123,7 +125,7 @@ void ObjectTransformWidget::create( const Box3f& box, const AffineXf3f& worldXf
 
     SceneRoot::get().addChild( controlsRoot_ );
     SceneRoot::get().addChild( activeLine_ );
-    controlsRoot_->setXf( worldXf );
+    setControlsXf( worldXf );
 
```

Comment:
please set default mode (Translation) in `create` or mb `better` reset function

Reply:
Fixed.